packages/main/src/ColorPalette.js

```diff
@@ -161,13 +196,43 @@ class ColorPalette extends UI5Element {
 		}
 	}
 
+	async chooseCustomColor(event) {
+		const colorPicker = await this.getColorPicker();
+		this._setColor(colorPicker.color);
+		this.closeDialog();
+	}
+
+	async closeDialog() {
+		const dialog = await this._getDialog();
+		dialog.close();
+	}
+
+	async _openMoreColorsDialog() {
```

Comment:
Just styling, perhaps you can make the naming more consistent.
 _openMoreColorsDialog is with "_", while chooseCustomColor and closeDialog - not.
_getDialog is with "_", while getColorPicker - not.